src/Handlebars.Net.Helpers/Helpers/EvaluateHelper.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using HandlebarsDotNet.PathStructure;
 using Stef.Validation;
 using HandlebarsDotNet.Helpers.Models;
-#if NETSTANDARD1_3_OR_GREATER || NET46_OR_GREATER
+#if NETSTANDARD1_3_OR_GREATER || NET46_OR_GREATER || NET6_0_OR_GREATER
```

Comment:
Could this not be simplified to #if !NETFRAMEWORK

Reply:
I don't think so because NET46 is defined as NETFRAMEWORK.